### DISCUSSION TITLE
Add Form Input Example

### DISCUSSION THREAD

**Discussion by MentalGear** [testing-library/testing-library-docs#1164] [Q&A]
Thx for this testing library - the docs are well written, accessibility is a priority and queries well explained.

However, when it comes to the manipulation of form elements, I couldn't find an example or section of it in the docs.

Doing this, for example, works, but gives a typescript error in VSCode

```ts
        const firstName = screen.getByRole('textbox', { name: /first name/i })
        firstName.value = 'John Simpleton'
```

`Property 'value' does not exist on type 'HTMLElement'`

**Comment by timdeschryver** [comment]
To interact with elements either use [fireEvent](https://testing-library.com/docs/dom-testing-library/api-events) or [user-event](https://testing-library.com/docs/user-event/utility#type)
